spring-aot/pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.kubernetes</groupId>
+    <artifactId>client-java-spring-aot-integration</artifactId>
+    <packaging>bundle</packaging>
+    <name>client-java-spring-aot-integration</name>
+    <url>https://github.com/kubernetes-client/java</url>
+    <parent>
+        <artifactId>client-java-parent</artifactId>
+        <groupId>io.kubernetes</groupId>
+        <version>16.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <dependencies>
+        <dependency>
+            <groupId>org.reflections</groupId>
+            <artifactId>reflections</artifactId>
+            <version>0.10.2</version>
```

Comment:
nit: move the versioned dependency to the parent pom and let the children modules inherit.

Reply:
good idea